src/sentry/static/sentry/app/views/settings/organizationRepositories/organizationRepositories.jsx

```diff
@@ -61,31 +63,45 @@ export default class OrganizationRepositories extends React.Component {
 
     return (
       <div>
-        <SettingsPageHeader
-          title={t('Repositories')}
-          action={
-            <DropdownLink
-              anchorRight
-              className="btn btn-primary btn-sm"
-              title={t('Add Repository')}
-            >
-              {repoConfig &&
-                repoConfig.providers &&
-                repoConfig.providers.map(provider => {
-                  return (
-                    <MenuItem noAnchor={true} key={provider.id}>
-                      <AddRepositoryLink
-                        provider={provider}
-                        orgId={orgId}
-                        onSuccess={onAddRepo}
-                      />
-                    </MenuItem>
-                  );
-                })}
-            </DropdownLink>
-          }
-        />
-
+        <Feature
+          feature={['bitbucket-integration', 'github-enterprise']}
+          requireAll={false}
+        >
+          {({hasFeature}) => {
+            return hasFeature ? (
+              <AlertLink to={`/settings/${orgId}/integrations/`}>
+                {t(
+                  'Want to add a repository to start tracking commits? Install or configure your version control integration here.'
+                )}
+              </AlertLink>
+            ) : (
+              <SettingsPageHeader
+                title={t('Repositories')}
+                action={
+                  <DropdownLink
+                    anchorRight
+                    className="btn btn-primary btn-sm"
+                    title={t('Add Repository')}
+                  >
+                    {repoConfig &&
+                      repoConfig.providers &&
+                      repoConfig.providers.map(provider => {
+                        return (
+                          <MenuItem noAnchor={true} key={provider.id}>
+                            <AddRepositoryLink
+                              provider={provider}
+                              orgId={orgId}
+                              onSuccess={onAddRepo}
+                            />
+                          </MenuItem>
+                        );
+                      })}
+                  </DropdownLink>
+                }
+              />
+            );
+          }}
+        </Feature>
```

Comment:
Oh wait, do we want to just completely be getting rid of the page header?

Reply:
uhhh i didn't think it mattered all that much but i can leave the header and just remove the button if you think that's better

Reply:
IMO it's better since it keeps this page more consistent. Even better better I think might be to *disable* the button with a tooltip